content/tutorials/remote_sensing_visualization/GRASS_remotesensing.qmd

```diff
@@ -0,0 +1,790 @@
+---
+title: "Remote Sensing With GRASS"
+author: "Michael Barton"
+date: 2025-02-14
+date-modified: today
+format: 
+  html:
+    embed-resources: true
+    toc: true
+    code-tools: true
+    code-copy: true
+    code-fold: false
+page-layout: article
+categories: [intermediate, advanced, GUI, imagery, remote sensing]
```

Comment:
```suggestion
categories: [beginner, GUI, imagery, remote sensing]
```

Reply:
Although I tried to make this as approachable as possible, I don't see this as something that a person first learning GIS (beginner) would be doing. I also listed "advanced" because it is also something that would be useful for an advanced GIS user who has not worked with remote sensing imagery before.

Reply:
If this is advanced, I am not sure what all the other tutorials on Python and R are. To me this is clearly targeted for beginners, it explains remote sensing etc. Think of people with basic GIS skills.